File: Pemrograman Berorientasi Objek/Materi/Pertemuan 3/Inheritance/java/Karyawan.java
class Karyawan extends Manusia{
	
	private String nomor_pegawai;
	private String jabatan;
	private String departemen;
	
	Karyawan(){
	}
	
	public void setNomorPegawai(String nomor){
		this.nomor_pegawai = nomor;
	}
	
	public String getNomorPegawai(){
		return this.nomor_pegawai;
	}
	
	public void setJabatan(String jabatan){
		this.jabatan = jabatan;
	}
	
	public String getJabatan(){
		return this.jabatan;
	}
	
	public void setDepartemen(String departemen){
		this.departemen = departemen;
	}
	
	public String getDepartemen(){
		return this.departemen;
	}
	
}

File: Pemrograman Berorientasi Objek/Materi/Pertemuan 3/Inheritance/cpp/Karyawan.cpp
class Karyawan : public Manusia{
	
	private:
		string nomor_pegawai;
		string jabatan;
		string departemen;
	
	public:
		Karyawan(){
		}
		
		void setNomorPegawai(string nomor){
			this->nomor_pegawai = nomor;
		}
		
		string getNomorPegawai(){
			return this->nomor_pegawai;
		}
		
		void setJabatan(string jabatan){
			this->jabatan = jabatan;
		}
		
		string getJabatan(){
			return this->jabatan;
		}
		
		void setDepartemen(string departemen){
			this->departemen = departemen;
		}
		
		string getDepartemen(){
			return this->departemen;
		}
		
		~Karyawan(){
		}
		
};